modules/nf-core/modules/cnvkit/batch/main.nf

```diff
@@ -1,6 +1,6 @@
 process CNVKIT_BATCH {
     tag "$meta.id"
-    label 'process_low'
+    label 'process_medium'
```

Comment:
this should not be changed here but either in the modules or just adapt the resource label in the pipeline config